src/ObjectUtils/ElfFile.cpp

```diff
@@ -54,16 +54,16 @@ class ElfFileImpl : public ElfFile {
   ElfFileImpl(std::filesystem::path file_path,
               llvm::object::OwningBinary<llvm::object::ObjectFile>&& owning_binary);
 
-  [[nodiscard]] ErrorMessageOr<ModuleSymbols> LoadSymbolsFromSymtab() override;
+  [[nodiscard]] ErrorMessageOr<ModuleSymbols> LoadSymbols() override;
```

Comment:
I think this becomes too vague.. maybe LoadDebugSymbols, and add a comment that for the elf-file it is .symtab section.

Reply:
I had this first as LoadSymbolsFromSymbolTable(), which also exists for COFF files. WDYT?

Reply:
seems like it has the same semantics - it is a debug symbols table.

Which is deprecated if I read this correctly:
https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#coff-file-header-object-and-image
"The file offset of the COFF symbol table, or zero if no COFF symbol table is present. This value should be zero for an image because COFF debugging information is deprecated."

Reply:
and dynamic symbols (or exported symbols) are coming from edata section.

Reply:
Ok, I'll go with LoadDebugSymbols to keep other options open (such as reading them from DWARF).